lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -51,6 +51,33 @@ def check_export_status(url, timeout: 180)
         assert_status_reponse_required_field(response_body)
 
         @output = response_body['output']
+
+        assert_output_has_type_url(@output)
```

Comment:
There's no need to pass an instance variable to an instance method (also in line 162). If the concern is being able to override the argument for tests, that can be done with a default argument:
```
def assert_output_has_type_url(output = @output)
```